investment-analysis-on-edgar-using-vss/edgar_data_pipeline/S3_utils.py

```diff
@@ -0,0 +1,177 @@
+import boto3
+import os
+import numpy as np
+import pandas as pd
+
+class S3Utils:
+    def __init__(self, s3_bucket):
+        """
+        Initializes the S3Utils class with the specified S3 bucket.
+
+        Args:
+        - s3_bucket (str): The name of the S3 bucket.
+
+        Initializes the S3 client using the 'benchmark' profile from the AWS credentials file.
+        """
+        self.s3_bucket = s3_bucket
+
+        # Initialize the S3 client
+        self.session = boto3.Session(profile_name='benchmark')
```

Comment:
We should not have the hardcoded profile name in here, maybe allow them to pass the profile in as a parameter as well as choosing the default profile if one is not passed in